main.go

```diff
@@ -90,7 +90,12 @@ func main() {
 	log.SetEnableDebugLog(cfg.VerboseLog)
 
 	log.Infof("Update homebrew")
-	cmd := command.New("brew", "update").SetStdout(os.Stdout).SetStderr(os.Stderr)
+	if cfg.VerboseLog {
+		cmd := command.New("brew", "update").SetStdout(os.Stdout).SetStderr(os.Stderr)
```

Comment:
Please define `cmd` outside the if-else scope, so it will be visible.

Reply:
Okay! Thanks! I'll update!